### DISCUSSION TITLE
Enable/Disable the interactivity of an element

### DISCUSSION THREAD

**Discussion by navidshad** [clientIO/joint#2316] [Q&A] [answered]
### Introduction

I was searching for interactivity properties of Jointjs elements and I figured out it handled by a Cellview for each element. then I got this code suggestion from copilot that works well. this line will deactivate the element move ability: 
```js
rectangle.findView(paper).options.interactive = { elementMove: false };
```

So, I want to read the `Cellview.options` reference to see which parameters are available for it but I can't find much information on the internet even in the Jointjs doc. do you know any reference link?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by jamesgeorgewilliams** [comment, marked as answer]
You can find the different options for Link & Element interactivity within the table in the following documentation:
https://resources.jointjs.com/docs/jointjs/v3.7/joint.html#dia.Paper.prototype.options.interactive
